source/opt/aggressive_dead_code_elim_pass.cpp

```diff
@@ -36,6 +36,7 @@ const uint32_t kLoopMergeContinueBlockIdInIdx = 1;
 }  // namespace
 
 bool AggressiveDCEPass::IsVarOfStorage(uint32_t varId, uint32_t storageClass) {
+  if (varId == 0) return false;
```

Comment:
Seems to merit an update to the method comment.

Reply:
Done.